libs/vm-modules/src/math/abs.cpp

```diff
@@ -44,6 +44,14 @@ fetch::math::meta::IfIsMath<T, T> Abs(VM * /*vm*/, T const &a)
   return x;
 }
 
+template <typename T>
+fetch::meta::EnableIfSame<T, Ptr<Fixed128>, T> AbsPtr(VM *vm, T const &a)
```

Comment:
might be nice to set up some vm-modules/meta/type-traits file to implement a:
IfIsPtrFixedPoint128

Reply:
on it now.

Reply:
fixed.